backend/src/services/group/group.service.ts

```diff
@@ -74,6 +75,49 @@ class Group {
 
     await this.#groupRepository.delete(id);
   }
+
+  public async update(
+    id: string,
+    { name, tenantId, workersIds, permissionsIds }: EAMGroupCreateRequestDto,
+  ): Promise<EAMGroupCreateResponseDto | null> {
+    const group = await this.#groupRepository.getGroupById(id);
+
+    if (!group) {
+      throw new EAMError({
+        status: HttpCode.NOT_FOUND,
+        message: ExceptionMessage.GROUP_NOT_FOUND,
+      });
+    }
+    if (name && name != group.name) {
```

Comment:
```suggestion
    if (name !== group.name) {
```
name should always be returned?